### ISSUE TITLE
Mantenimiento-->Equipo-->Ordenar: error de funcionalidad en los botones de orden 

### ISSUE THREAD

**Issue by floto-trazalog** [Trazalog/traz-prod-assetplanner#70]
Ambiente test - v2.5

**Descripción**
Los botones de orden en la tabla de registros de los equipos pierden funcionalidad

![image](https://user-images.githubusercontent.com/88331456/212902729-7a8a8d3c-e7d7-4209-af08-49aba30e804f.png)

**Comment by fhernandez-trazalog** [comment]
Efectivamente, sólo dice "Procesando..." brevemente. Pero no efectúa el orden solicitado.
